packages/core/contracts/financial-templates/optimistic-distributor/OptimisticDistributor.sol

```diff
@@ -181,13 +174,14 @@ contract OptimisticDistributor is Lockable, MultiCaller, Testable {
         // Store funded reward and log created reward.
         Reward memory reward =
             Reward({
-                distributionProposed: DistributionProposed.None,
+                distributionExecuted: false,
                 sponsor: msg.sender,
                 rewardToken: rewardToken,
                 maximumRewardAmount: maximumRewardAmount,
                 earliestProposalTimestamp: earliestProposalTimestamp,
                 optimisticOracleProposerBond: optimisticOracleProposerBond,
                 optimisticOracleLivenessTime: optimisticOracleLivenessTime,
+                blockingProposalTimestamp: 0,
```

Comment:
can we think about better names for this? what it really is is the dispute time, as defined on `        rewards[rewardIndex].blockingProposalTimestamp = timestamp;`. would `disputeTimestamp` be better? why is it a "blockingProposal", if its done via a dispute?

Reply:
This is not dispute timestamp, but the time when previous request/proposal was made through OD (or 0 if this is the first proposal). `_noBlockingProposal` then checks it and if non-0 asks OO `getRequest` with this timestamp. From OO perspective it is previous request timestamp, but from OD perspective it is previous proposal that matters (it just happens to be at the same timestamp when prior OO request was made). So maybe better alternative to use `previousProposalTimestamp` here?